### PR TITLE
[NPS-556] Move scroll-up button to avoid overlapping with Zendesk Widget

### PR DESCRIPTION
**Pull request by patpilus**
image

### CODE REVIEW

**Review by bartosz-zasada — APPROVED** [review]

File: src/Resources/theme.json [hunk 1]
```diff
@@ -237,7 +237,7 @@
                     "de-DE": "Wird über einem Viewport von 991px angezeigt"
                 },
                 "type": "media",
-                "value": "app/storefront/src/assets/logo/logo_header_ver3.gif",
+                "value": "app/storefront/src/assets/logo/logo_header_ver3_desktop.gif",
```
Comment (bystro) [inline src/Resources/theme.json:240]: what were the reasons of .gif filenames change?

Comment (patpilus) [reply src/Resources/theme.json:240]: After update to shopware 6.1.4 there were issues with duplicated values in `theme.json` file during `theme:dump` command. @bartosz-zasada asked me to push this change.

File: src/Resources/views/storefront/layout/scroll-up.html.twig [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% sw_extends '@Storefront/storefront/layout/scroll-up.html.twig' %}
+
+{% block layout_scroll_up_button %}
+    <div class="scroll-up-button js-scroll-up-button mb-5">
```
Comment (bystro) [inline src/Resources/views/storefront/layout/scroll-up.html.twig:4]: so adding the bootstrap mb-5 css classname is the change avoiding scroll-up button overlapping with Zendesk Widget ?

Comment (patpilus) [reply src/Resources/views/storefront/layout/scroll-up.html.twig:4]: yes, it's bootstrap class responsible for margin bottom: https://getbootstrap.com/docs/4.4/utilities/spacing/

**Review by bystro — APPROVED** [review]

**Pull request merged**